src/test/java/in/projecteka/consentmanager/link/link/LinkUserJourneyTest.java

```diff
@@ -571,4 +575,117 @@ public void shouldFailOnLinkCareContexts() throws Exception {
                 .expectStatus().is5xxServerError();
     }
 
+    @Test
+    public void shouldGetPatientLinkReference() throws IOException {
+        var token = string();
+        var patientLinkReferenceRequest = patientLinkReferenceRequest().build();
+        var linkReferenceRequest = TestBuilders.linkReferenceRequest().build();
+        var hipId = "10000005";
+        var linkReferenceResult = "{\n" +
```

Comment:
Please use the testBuilder instead of string.

Reply:
Done